src/repr/scalar/mod.rs

```diff
@@ -728,7 +723,8 @@ impl fmt::Display for ColumnType {
     }
 }
 
-/// Inlined from [`sqlparser::ast::Interval`] so that we can impl deserialize, ord
+/// An interval of time meant to express SQL intervals. Obtained by parsing an
+/// `INTERVAL '<value>' <unit> [TO <precision>]`.
```

Comment:
tiny nit: rustdoc and some IDEs distinguish between headers and content in terms of what's shown on different pages, so this could use a couple newlines between "intervals." and "Obtained".